### ISSUE TITLE
Show entire item name when selected

### ISSUE THREAD

**Issue by superluig164** [files-community/Files#14064]
### What feature or improvement do you think would benefit Files?

In File Explorer, when you select an item with a name that exceeds the length visible, the entire name is shown (in thumbnail views).  I believe this would be a good feature to extend to Files, albeit a bit more elegantly.  My proposal is that, when a such an item is selected, its selection box would become solid (rather than an outline) and extend over the item below it, revealing the entire item name.  Also, I think this functionality could also be extended to list views, similarly extending the box over the item below it, putting the rest of the name on a new line.  Of course, clicks should go right through to the item below, in case the user wants to select that item, kind of like a persistent tooltip.

### Requirements

- Use solid selection boxes rather than outlines (either sometimes or always)
- Occasionally cover items with the full names of items above them, if those names are long.

### Files Version

3.0.8.0

### Windows Version

10.0.22631.2428

**Comment by Josh65-2201** [comment]
Thanks for the feedback, I've added this to the project board.

**Comment by superluig164** [comment]
I thought of an edge case which you might want to keep note of - if multiple items are selected, perhaps each item could check the one below it.  If the item below it is also selected, _don't_ extend the box, but if the item below isn't selected, extend the box as normal.  That way all selected items could benefit from the feature without obscuring other selected items.

**Comment by yaira2** [comment]
I've personally found this behavior in File Explorer to be a little messy as it makes the item sizes inconsistent, instead I prefer hovering over the item for the tooltip.

**Comment by superluig164** [comment]
I don't think it should change the size of the rows, I actually really hate that too.  I was only saying _while the item is selected_, this way it keeps the consistent grid without losing the ability to easily see the full name of the file without waiting for the tooltip or using the details pane.

**Comment by bgen5pax** [comment]
> I've personally found this behavior in File Explorer to be a little messy as it makes the item sizes inconsistent, instead I prefer hovering over the item for the tooltip.

Do yo plan to implement "tooltip on hover"?

**Comment by Josh65-2201** [comment]
Files already does.
